### ISSUE TITLE
Unstated light dependencies

### ISSUE THREAD

**Issue by hugbubby** [unix121/i3wm-themer#24]
I'm counting xfce4-battery-plugin, networkmanager, nm-applet, clipit, i3exit....

**Comment by unix121** [comment]
Yeah, but those will not cause i3wm to crash if they are not installed. I could add them as dependencies in the README.md and in the script later, but I'm not that sure if it's worth the extra big README.md at this time.

I think a good thing would be to add a new bash script with the 'extras' for people to run to get those.

**Comment by kra3** [comment]
please, add.. same goes for the script (`i3-exit`) used to suspend/shutdown/restart etc actions.
Either include it in the repo or provide a link to it, please
